.travis.yml

```diff
@@ -149,7 +149,7 @@ matrix:
       before_script:
         - . ./ci/travis/ci.sh build
       script:
-        - travis_wait 90 bazel test --config=ci --test_output=errors --build_tests_only --test_tag_filters=learning_tests_tf rllib/...
+        - travis_wait 90 bazel test --config=ci --test_output=streamed --build_tests_only --test_tag_filters=learning_tests_tf rllib/...
```

Comment:
@sven1977 we shouldn't use errors only, since that means a timeout will completely hide which test was hung.